lib/include/jxl/decode.h

```diff
@@ -422,7 +422,15 @@ JxlDecoderSetKeepOrientation(JxlDecoder* dec, JXL_BOOL keep_orientation);
  * requires more JxlDecoderProcessInput calls to continue.
  *
  * @param dec decoder object
- * @return JXL_DEC_SUCCESS when decoding finished and all events handled.
+ * @return JXL_DEC_SUCCESS when decoding finished and all events handled. If you
```

Comment:
Could we continue to return `JXL_DEC_SUCCESS` if no new input data is provided and there are no more new events (and we happen to be at a box boundary)? That would simplify the logic a bit and make `kFinished` stage mean that the jxl codestream reached the end (but other boxes are still ok).

Reply:
Good idea, done